### DISCUSSION TITLE
Unable to Sign In to OpenGWAS Due to Server Error

### DISCUSSION THREAD

**Discussion by yuka312** [MRCIEU/TwoSampleMR#579] [Q&A]
Hi. 
To use TwoSampleMR, signing in and issuing a token are required.
However, I am unable to connect to the server.

When I attempt to sign in using my university's Microsoft account, the following error message appears:
"Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."
I have waited and retried multiple times, but the issue persists.
Additionally, when I try to sign in using my GitHub account, the following error message appears after entering my name:
"list indices must be integers or slices, not str"
Could you kindly advise me on how I can successfully sign in to OpenGWAS? This issue is quite urgent as it relates to my graduation research, and I would greatly appreciate your assistance at your earliest convenience.

Thank you very much for your time and support.

**Comment by jingotu** [comment]
Has your problem been resolved yet?
My code was working well until this week... I think they're having a maintenance downtime since their API page (https://gwas-api.mrcieu.ac.uk/) has been showing Error 503 these days.  
Does anyone know what is actually going on?

![image](https://github.com/user-attachments/assets/b8314417-4aec-48b7-ab3a-6ae90c784720)
